Source/WebCore/html/ImageBitmap.h

```diff
@@ -152,8 +164,11 @@ class ImageBitmap final : public ScriptWrappable, public RefCounted<ImageBitmap>
     static void createFromBuffer(ScriptExecutionContext&, Ref<ArrayBuffer>&&, String mimeType, long long expectedContentLength, const URL&, ImageBitmapOptions&&, std::optional<IntRect>, ImageBitmapCompletionHandler&&);
     void updateMemoryCost();
 
-    std::optional<ImageBitmapBacking> m_backingStore;
+    RefPtr<ImageBuffer> m_bitmap;
     std::atomic<size_t> m_memoryCost { 0 };
+    const bool m_originClean : 1 { false };
+    const bool m_premultiplyAlpha : 1 { false };
+    const bool m_forciblyPremultiplyAlpha : 1 { false };
```

Comment:
It is unfortunate to repeat the same flags here and in `DetachedImageBitmap`

Reply:
I'm not sure how much of a problem it is,  having these duplicated in the DetachedImageBitmap and here. 
Eventually I'd like us to preserve the premultiplication status in the native image, thus we would get rid of m_premultiplyAlpha and especially m_forciblyPremultiplyAlpha